lib/Input.js

```diff
@@ -0,0 +1,32 @@
+
+const minimist = require('minimist');
+
+class Input {
+    constructor() {
```

Comment:
constructor(arr) {
    const { type, payload } = parse(arr);
    this.type = type;
    this.payload = payload;
  }